app/assets/stylesheets/screens/_papers.scss

```diff
@@ -354,3 +354,9 @@ html.control-bar-sub-nav-active #paper-body .oo-ui-toolbar {
     }
   }
 }
+
+.placeholder {
+  background: #aaa !important;
```

Comment:
Color literals like `#aaa` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used